src/main/java/jsr223/cpython/PythonScriptEngine.java

```diff
@@ -65,8 +65,21 @@
 
     private PythonProcessBuilderUtilities processBuilderUtilities = new PythonProcessBuilderUtilities();
 
-    public PythonScriptEngine() {
+    private static final String log4jConfigurationFile = "config/log/scriptengines.properties";
 
+    public PythonScriptEngine() {
+        try {
+            org.apache.log4j.PropertyConfigurator.configure(getClass().getClassLoader()
```

Comment:
normally the forked JVM when executing a task is already configured for log4j. I am not sure it is safe to redo the log4j configuration inside the python script engine